### ISSUE TITLE
Changing value of the tag

### ISSUE THREAD

**Issue by mpele** [topobyte/osm4j#4]
Sorry, I understand that this is not an issue but I need some help.

Is it possible to change the value of the tag in the entity during iterating? I would like to redefine tag name but I couldn't find the way to do it. If it is not possible, is it possible to get the raw data for the entity?

**Comment by mpele** [comment]
Thank you for your quick response - I managed to do tag manipulation :)

May I suggest to make example how tag manipulation could be done for other potential users.

**Comment by sebkur** [comment]
yes, good idea. May I ask what your use case is? Maybe it can be inspiration for an example...

**Comment by mpele** [comment]
Uh, it is very complicate to explain... 
Let's say it very simple - I am preparing data for import in tile server, but I do not want (I am not able) to change carto style because it depends on the country, so I am adjusting data before import. 

ps. Is your library able to load the poly files and check if some point is in the polygon?

**Comment by akamuza** [comment]
I have the same task. Use case: augmenting pbf files with some custom extra tags before importing this pbf into graphhopper.
E.g. I'd like to mark roads with custom tag that they belong to big cities polygons. This allows to lower priority of this roads for intercity logistics.
